src/markspelling.py

```diff
@@ -44,6 +44,9 @@ def __init__(self, pwl, language='en_GB', errortotalprev=0):
         self.regexhtmlclean = re.compile(r'\`.+?\`')
         self.regexlink = re.compile(r'\[(.+?)\]\(.+?\)')
```

Comment:
There doesn't seem to be a test line for this.

Reply:
There is now!